guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,7 +4,20 @@
 [id="foreman-headline-features"]
 == Headline Features
 
-There are no highlights with Foreman {ProjectVersion}.
+=== IPv6 Support
+
+Deployment and operation in IPv6-only networks is now fully supported.
+
+Provisioning over IPv6 is supported on bare metal hosts.
+For compute resources, you can define the machine outside Foreman on the compute resource and then you can provision the machine as bare metal in Foreman.
+
+=== Secure Boot provisioning
+
+Support for Secure Boot provisioning on bare-metal, VMware vSphere and Libvirt across multiple operating systems.
+
+=== Invalidate JWT for global registration
+
+This will give users the ability to edit user permissions and invalidate JWT tokens for other users, invalidate self's token, and admins can invalidate selfs and other user tokens
```

Comment:
In release notes you are talking to the user, not about the user. When you write "This will give users" you're talking in the future tense, but the change has already landed so it should be written in the present tense.

The exact phrasing can probably be best reviewed by a writer, but how about:

```suggestion
Users can now invalidate their own tokens for global registration.
Users with the `edit_users` permission can also invalidate all tokens for all users in a single action.
```